pandas/core/frame.py

```diff
@@ -1043,6 +1041,56 @@ def _repr_html_(self) -> str | None:
         else:
             return None
 
+    @overload
+    def to_string(
+        self,
+        buf: None = None,
```

Comment:
can leave out the defaults for the overloads. It's only the parameter types that are relevant in overloads.

```suggestion
        buf: None = ...,
```